common/presenters/moves-to-csv.js

```diff
@@ -169,14 +169,29 @@ const documents = [
   },
 ]
 
-module.exports = function movesToCSV(moves) {
+const additional = [
+  {
```

Comment:
This might be nicer group with the `move` items. Probably makes most sense appearing after `date` on line 125. Then the CSV would read `date` -> `time_due` which feels logical.

Reply:
Unfortunatley there are users relying on the existing set order of the columns